client/my-sites/stats/stats-list/action-link.jsx

```diff
@@ -1,55 +1,32 @@
 /**
  * External dependencies
  */
-import React, { PureComponent, PropTypes } from 'react';
```

Comment:
What happened to this file? A bad rebase conflict resolution? Seems like it reverses positive refactorings.